scripts/spack/packages/tribol/package.py

```diff
@@ -142,7 +142,7 @@ class Tribol(CachedCMakePackage, CudaPackage, ROCmPackage):
     depends_on("python", when="+devtools")
     depends_on("py-shroud", when="+devtools+fortran")
     depends_on("py-sphinx", when="+devtools")
-    depends_on("llvm+clang@10.0.0", when="+devtools", type="build")
+    depends_on("llvm+clang@14:", when="+devtools", type="build")
```

Comment:
```suggestion
    depends_on("llvm+clang@14", when="+devtools", type="build")
```

i would recommend this as major clang-format versions produce different style